.github/workflows/ci.yml

```diff
@@ -0,0 +1,334 @@
+name: 'CI'
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+env:
+  APPLICATION_ENV: ci.pgsql
+  SPRYKER_TESTING_ENABLED: 1
+  APPLICATION_STORE: DE
+  PROJECT: suite
+  POSTGRES_PORT: 5433
+  POSTGRES_PASSWORD: postgres
+  POSTGRES_USER: postgres
+  POSTGRES_DB: DE_test_zed
+  MYSQL_PASSWORD: secret
+  MYSQL_USER: root
+  MYSQL_DATABASE: DE_test_zed
+
+jobs:
+  validation-7-4:
+    name: 'Static analysis / PHP 7.4'
+    runs-on: ubuntu-18.04
+    services:
+      postgres:
+        image: postgres:12
```

Comment:
we use mariadb as default DB. In this case I guess default checks should use it too

Reply:
Renamed steps. Now it looks like checks for MariaDB are default.